lib/cc/engine/analyzers/violation.rb

```diff
@@ -8,8 +8,8 @@ module Analyzers
       class Violation
         attr_reader :issue
 
-        def initialize(base_points, issue, hashes)
-          @base_points = base_points
+        def initialize(language, issue, hashes)
+          @language = language
```

Comment:
Would calling this either `strategy` or `language_strategy` better match the variable [passed in](https://github.com/codeclimate/codeclimate-duplication/pull/67/files#diff-d285873a9a1a70964311d140233c6dfaR65)?